WordPress/build.gradle

```diff
@@ -461,8 +464,8 @@ dependencies {
     kapt project(':libs:WordPressProcessors')
 
     // Debug
-    debugImplementation 'com.facebook.stetho:stetho:1.5.1'
-    debugImplementation 'com.facebook.stetho:stetho-okhttp3:1.5.1'
+    debugImplementation 'com.facebook.stetho:stetho:1.6.0'
```

Comment:
The only change in the new version is that it uses AndroidX - https://github.com/facebook/stetho/releases